docs/DafnyRef/Plugins.md

```diff
@@ -148,11 +155,66 @@ In that case, we could return:
   };
 ```
 
+#### 15.1.2.2. Request handler plugin
+
+The request handler will be simple: the request will get a `TextDocumentIdentifier` as parameter and will return a `bool` value denoting whether the provided `DocumentUri` has any `LoopStmt`'s in it.
+
+Then, open the file `MyPlugin.cs`, remove everything, and write the imports and a namespace:
+
+```csharp
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Server;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using Microsoft.Dafny.LanguageServer.Plugins;
+using Microsoft.Dafny.LanguageServer.Workspace;
+using MediatR;
+using Microsoft.Dafny;
+
+namespace MyPlugin;
+```
+
+After that, add a `PluginConfiguration` that will add all the request handlers of your plugin.
+This class will be discovered and instantiated automatically by Dafny.
+```csharp
+public class TestConfiguration : PluginConfiguration {
+  public override LanguageServerOptions WithPluginHandlers(LanguageServerOptions options) {
+    return options.WithHandler<DummyHandler>();
+  }
+}
+```
+Note that you could also override the methods `GetRewriters()` and `GetCompilers()` for other purposes, but this is out of scope for this tutorial.
```

Comment:
```suggestion
```
These two methods are already mentioned above and I find it breaks the flow of the tutorial